lib/whedon/processor.rb

```diff
@@ -152,6 +153,14 @@ def citation_string
       return "#{paper.citation_author}, (#{paper_year}). #{paper.plain_title}. #{ENV['JOURNAL_NAME']}, #{paper_volume}(#{paper_issue}), #{paper.review_issue_id}, https://doi.org/#{paper.formatted_doi}"
     end
 
+    def authors_map
```

Comment:
I think if it is going to be a nested hash, then a different form of encoding the URL should be done, specifically, I think the hash should be converted to JSON or YAML and sent as a string. This will simplify converting back to the hash on the JOSS side. Using `URI.encode_www_form` on the nested hash and then decoding with `URI.decode_www_form` converts the nested hash to a string representation, that then has to be (safely) parsed, and the simplest way to do that seems to be with the JSON module (see [this SO answer](https://stackoverflow.com/a/22820407)). So we might as well just use the JSON (or YAML) module from the start, depending on what's already loaded (if anything) over at JOSS.

I rebased onto `master` and pushed this change. Let me know what you think!